model/src/cfd.rs

```diff
@@ -1425,6 +1425,8 @@ impl Cfd {
 
         let hours_left = settlement_time - OffsetDateTime::now_utc();
 
+        tracing::trace!(time_left_in_cfd = %hours_left, "Calculating hours to extend in rollover");
```

Comment:
```suggestion
        tracing::trace!(target="rollover",time_left_in_cfd = %hours_left, "Hours to extend in rollover");
```

Not sure if `rollover` is a good target as per the definition of how `target` is to be used. But given that this is a `trace` log I think we need a target because we don't want all logs on trace.

Reply:
> Not sure if rollover is a good target as per the definition of how target is to be used. But given that this is a trace log I think we need a target because we don't want all logs on trace.

Good point. I'll fix this.

> Additionally, as far as I understand the code, this is already calculated at this point.

`time_left_in_cfd != hours_to_extend`.